packages/api/src/controllers/animalIdentifierPlacementController.js

```diff
@@ -0,0 +1,39 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import AnimalIdentifierPlacement from '../models/animalIdentifierPlacementModel.js';
+
+const animalIdentifierPlacementController = {
+  getAnimalIdentifierPlacements() {
+    return async (req, res) => {
+      try {
+        const { default_type_id } = req.query;
```

Comment:
Just clarifying it won't be possible to assign an identifier placement to custom_type_id

Reply:
Yeah, it's not too clear yet how we will handle identifier placement for custom types (there's an ongoing Slack convo) so for now I've left that out here entirely